client/condition.go

```
@@ -87,17 +87,31 @@ func (c *equalityConditional) Matches() (map[string]model.Model, error) {
 func (c *equalityConditional) Generate() ([][]ovsdb.Condition, error) {
 	models, _ := c.Matches()
```

I would probably not ignore this error now we have opened the API of `RowsByModels` to return one.

---

@jcaamano I just reworked the commit to respect the errors here and test for them in api_test.go.